src/emotion/Button.tsx

```diff
@@ -0,0 +1,62 @@
+/* @jsx jsx */
+
+import { jsx } from '@emotion/core';
+import { HTMLAttributes, SFC } from 'react';
+
+import { Box, IBox } from './Box';
+import { useTheme } from './theme';
+
+export const Button: SFC<IButton> = props => {
+  const { as = Box, disabled, ...rest } = props;
+
+  const css = [...buttonStyles({ disabled })];
```

Comment:
Custom styles are ignored here I guess :)

Reply:
Nope! Because it just gets passed through to our foundational component (Box).

Reply:
@marbemac assuming that the user doesn't change the `as` prop and do something like
`<Button as="div" />`